src/tests/ftest/io/dfuse_bash_cmd.py

```diff
@@ -92,7 +93,8 @@ def test_bashcmd(self):
                                                abs_file_path2),
                             "ls -al {}".format(abs_file_path2),
                             "rm {}".format(abs_file_path2),
-                            "rmdir {}".format(abs_dir_path)]
+                            "rmdir {}".format(abs_dir_path),
+                            "echo 12345 > {0}/test; cat {0}/test".format(mount_dir)]
```

Comment:
Minor: Will be nice to have one command each line to be consistant with the test structure. So something like this:
`"echo 12345 > {}/test".format(mount_dir),`
`"cat {}/test".format(mount_dir)]`